recipe/meta.yaml

```diff
@@ -36,6 +39,8 @@ about:
   license: MIT
   license_file: LICENSE.txt
```

Comment:
No license family specified

Reply:
The `license_family` was added to the recipe

https://github.com/AnacondaRecipes/qtawesome-feedstock/blob/update_to_1.0.3/recipe/meta.yaml#L44